go/chat/server.go

```diff
@@ -2070,7 +2087,106 @@ func (h *Server) UpdateTyping(ctx context.Context, arg chat1.UpdateTypingArg) (e
 	return nil
 }
 
+func (h *Server) checkInConv(ctx context.Context, convID chat1.ConversationID) (in bool, err error) {
+	h.Debug(ctx, "+ checkInConv(%v)", convID)
```

Comment:
I would just use `GetUnverifiedConv` for this guy, it does most of the work for you. You don't need to localize the conversation to see if you are in it. You just need the `chat1.Conversation` type.